terraform-aws-vpc/main.tf

```diff
@@ -53,7 +53,7 @@ resource "aws_internet_gateway" "main" {
 
 resource "aws_eip" "nat" {
   count = 2
-  vpc      = true
+  vpc   = true
 }
```

Comment:
Please separate `count` from the rest of the arguments with a newline.
Please fix it here and everywhere else where count is used